subscribe.go

```diff
@@ -163,6 +163,10 @@ func retrieveLastEventID(r *http.Request, logger Logger) string {
 		return id
 	}
 
+	if opt.protocolVersionCompatibility == 0 || opt.protocolVersionCompatibility >= 8 {
```

Comment:
```suggestion
	if opt.protocolVersionCompatibility != 0 || opt.protocolVersionCompatibility < 8 {
```
we could do the opposite here and move the next if statement inside ( not to duplicate `return ""` )

Reply:
I improved the logic to provide better debug information.